### PR TITLE
Refactor and mobile

### PR DESCRIPTION
**Pull request by ZeroSoulEater**
I added some CSS so it will look on mobile better according  to #20 

To achieve this I added some classes to the components and added some media queries in the css file

### CODE REVIEW

File: src/components/Container.js [hunk 1]
```diff
@@ -14,11 +14,13 @@ import {
 } from '../components';
 
 const styles = theme => ({
+
   button: {
-    margin: theme.spacing.unit,
+    //margin: theme.spacing.unit,
```
Comment (arshadkazmi42) [inline src/components/Container.js]: Let's not have commented code in the codebase.
If we don't need it, let's just remove it

File: src/components/Container.js [hunk 2]
```diff
@@ -14,11 +14,13 @@ import {
...
+
   },
   input: {
-    display: 'none',
+    //display: 'none',
```
Comment (arshadkazmi42) [inline src/components/Container.js]: Same as above

File: src/components/Container.js (hunk 1 above)
Comment (ZeroSoulEater) [reply src/components/Container.js]: Done :)

File: src/components/Container.js (hunk 2 above)
Comment (ZeroSoulEater) [reply src/components/Container.js]: Done

**Comment by ZeroSoulEater** [comment]
Should be 2 spaces now!

**Pull request merged**